dbms/src/Storages/KVStore/tests/gtest_region_persister.cpp

```diff
@@ -567,15 +567,19 @@ try
 
         auto region = makeRegion(createRegionMeta(region_100, table_id));
         TiKVKey key = RecordKVFormat::genKey(table_id, region_100, diff++);
-        region->insert(ColumnFamilyType::Default, TiKVKey::copyFrom(key), TiKVValue("value1"));
-        region->insert(ColumnFamilyType::Write, TiKVKey::copyFrom(key), RecordKVFormat::encodeWriteCfValue('P', 0));
-        region->insert(
+        region->insertFromSnap(tmt, ColumnFamilyType::Default, TiKVKey::copyFrom(key), TiKVValue("value1"));
```

Comment:
why use `insertFromSnapshot` rather than `insertDebug` for these test cases?

Reply:
Due to convenience. But yes, it is confusing, I will change it to debug